packages/react-api/src/api/SQL.js

```diff
@@ -41,16 +50,23 @@ export const executeSQL = async (credentials, query, opts = { format: '' }) => {
  * Create an 'SQL query' request
  * (using GET or POST request, depending on url size)
  */
-function createRequest({ credentials, query, opts }) {
+function createRequest({ credentials, connection, query, opts = {} }) {
```

Comment:
ok, I see opts = {} 👍🏻

Reply:
Yes, I didn't know why it was removed :).

We've removed the format because in cloud native opts allow to override all the parameters of each cloud provider. besides format we've timeoutms, maxBytesProcess, maxRows, dryRun, etc...